AppDashboard/lib/app_dashboard_data.py

```diff
@@ -647,3 +651,65 @@ def can_upload_apps(self):
     except Exception as err:
       logging.exception(err)
       return False
+
+  def get_dash_layout_settings(self, email=None):
+    """ Queries the UserAppServer to see what settings the user has saved
+    for customizing the UI of the Dashboard
+
+    Returns:
+      A JSON string describing the customization layout
+    """
+    default_dash_layout = '''
+        {"nav" :
+              {"App Management":{"True":
+                {"Upload App":"/apps/new",
+                "Delete App":"/apps/delete"}},
+              "AppScale Management":{"True":
+                {"Cloud Status":"/status/cloud",
+                "Manage Users":"/authorize"}},
+              "Debugging/Monitoring":{"True":
+                {"Monit":"temp1",
+                "TaskQueue":"temp2",
+                "Logging":"/logs"}}}
```

Comment:
Right. I'm not suggesting managing it in the template. You can remove it (or include it) by testing it against those variables before rendering the template. In my opinion, it's best to keep logic out of the template whenever possible.